modcc/printer/cudaprinter.cpp

```diff
@@ -252,4 +362,106 @@ void emit_common_defs(std::ostream& out, const Module& module_) {
     out << popindent << "};\n\n";
 }
 
+void emit_api_body_cu(std::ostream& out, APIMethod* e, bool is_point_proc) {
+    auto body = e->body();
+    auto indexed_vars = indexed_locals(e->scope());
+
+    if (!body->statements().empty()) {
+        out << "int tid_ = threadIdx.x + blockDim.x*blockIdx.x;\n";
+        out << "if (tid_<n_) {\n" << indent;
```

Comment:
We use such comments on closing `}' for namespaces, e.g.`}  // namespace arb`.
The trailining `}` for namespaces can be quite confusing, particularly if you don't indent namespaces.

Here the braces are usually quite close together, and are pretty easy to match up (look for matching `popindent` for each `indent`.)

Reply:
As we don't typically indent within the namespace scopes, it becomes more important to clarify where that lonely random closing brace is coming from. One can otherwise though follow it through the indentation, which IMO means it is unnecessary in most other contexts.